src/mapgen.cpp

```diff
@@ -199,27 +199,86 @@ void Mapgen::updateHeightmap(v3s16 nmin, v3s16 nmax)
 	//printf("updateHeightmap: %dus\n", t.stop());
 }
 
+inline bool Mapgen::updateLiquidHelper(int i, const v3s16 &em)
```

Comment:
Hrmm?  What does this function 'help' with?
What is `i` here?
What is `em`?
Why bother passing a const ref to em?  It's the size of a machine word on the most common platform Minetest will be run on.